Add log file to store player moves and scores

jayeshs999 opened the pull request:
TH elog file is in  the following format for each game-:
NG/   # start of game
{RoundNumber},{Type of value in last column-M/S},{Player Token},{Move/Score}
........
.........
.......
/NG # end of game

jayeshs999 committed:
Add log file to store player moves and scores 

TH elog file is in  the following format for each game-:
NG/   # start of game
{RoundNumber},{Type of value in last column-M/S},{Player Token},{Move/Score}
........
.........
.......
/NG # end of game

cheekujodhpur commented on game/core.py:
```diff
@@ -74,7 +75,9 @@ def _broadcast_message(self, message, typ='NORMAL'):
         if state is State.GAME_BEGIN:
-            return None, State.ROUND_BEGIN
+        	self.log_file.write("NG/"+"\n")
+        	self.log_file.write("Round,Type,Player,Move/Score"+"\n")
```
Why is the "\n" separate? Ideally, we never try to add strings

cheekujodhpur reviewed (commented):
Generally,

1. Use pylint or flake8 to lint-fix your code. Google what "lint" is.
2. Just a thought, maybe the logging can be merged with the `_broadcast_message` function?

jayeshs999 committed:
Correct styling and add player hand and top card info

Updated entry format-:
NG/
Round,Type,Player,Move/Score,TopCard(BeforeMove),PlayerHand(AfterMove)
/NG

cheekujodhpur commented:
Looks good.
